levels/level-0.md

```diff
@@ -0,0 +1,61 @@
+# Brukergrensesnittutvikler, nivå 0
+
+## Tittel
+
+Brukergrensesnittutvikler / User Interface Developer
+
+## Beskrivelse
+
+Den ansatte skal som regel jobbe i prosjekt sammen med én eller flere andre brukergrensesnittutviklere, og skal få hjelp og veiledning fra disse.
+
+Det forventes deltagelse i planlegging og gjennomføring av utvikling av gjenbrukbare enkeltkomponenter og komponentsystemer.
+
+## Erfaring
+
+Relevant utdannelse på bachelornivå
+
+_eller_
+
+Tilsvarende ferdighetsnivå
+
+## Faglig kompetanse
+
+Gode skriftlige og muntlige norsk- og engelskkunnskaper
+
+- Må kunne kommunisere med kunder om systemer og med kollegaer om fag
+
+Grunnleggende forståelse for, og erfaring med klientsideprogrammering og -teknologi (HTML/CSS/JS)
+
+- Må kunne lage nettsider ved bruk av disse teknologiene og kunne forklare hva de er for noe og hvorfor vi trenger dem
+
+Grunnleggende forståelse for HTTP
+
+- Hva er det vanligste verbene og hva brukes de til?
+
+Grunnleggende forståelse for bruk av eksterne APIer over HTTP
+
+- Hva er `fetch` og hvordan brukes det?
+- Hva er CORS og hvorfor finnes det?
+- Hva er en proxy og hvorfor brukes det?
```

Comment:
Er dette veldig relevant for utvikling av grensesnitt for episerver-sider?

Reply:
I see it as a question of fartstid i faget, if you've been around just a bit in the right contexts.

Reply:
And do we really want to tie our skills to Episerver? (The answer is "no").

Reply:
Perhaps generalize to something about "having some familiarity with "mv**"